tree/dataframe/src/RCsvDS.cxx

```diff
@@ -68,6 +71,11 @@ double-quote characters must be represented by a pair of double-quote characters
 The current implementation of RCsvDS reads the entire CSV file content into memory before
 RDataFrame starts processing it. Therefore, before creating a CSV RDataFrame, it is
 important to check both how much memory is available and the size of the CSV file.
+
+RCsvDS can handle empty cells and also allows the usage of the special keywords "NaN" and "nan" to
+indicate empty cells. Unless a column is specified to be of type string, the special keywords will be
+read as empty cells. If the column is of type double, empty cells are stored internally as `nan`.
+If the column if of type Long64_t/bool empty cells are stored as 0/false.
```

Comment:
Nice, really good to mention the new behavior this way. Maybe it's just me but I think we should say that empty cells are treated as nans (which have a well-defined corresponding value for floating point and integer values) rather than that nans are treated as empty cells (which then requires reading the next sentence about how empty cells are treated to be understood).

Reply:
I updated it. As a note - only doubles have `nan`.

Reply:
It now reads:
```
RCsvDS can handle empty cells and also allows the usage of the special keywords "NaN" and "nan" to
indicate `nan` values. If the column is of type double, these cells are stored internally as `nan`.
Empty cells and explicit `nan`-s inside columns of type Long64_t/bool are stored as zeros.
```